models/company.py

```diff
@@ -0,0 +1,52 @@
+from db import db
+# import enum
+#
+#
+# class BusinessChoices(enum.Enum):
+#     finance = 'finance'
+#     health_care = 'health care'
+#     education = 'education'
+
+class CompanyModel(db.Model):
+    __tablename__ = 'company'
+
+    id = db.Column(db.Integer, primary_key=True)
+    us_employer_id = db.Column(db.String(50))
+    company_name = db.Column(db.String(200))
+    # business_area = db.Column(db.Enum(BusinessChoices), nullable=False)  # ******* this should be enum *******
+    business_area = db.Column(db.String(50), nullable=False)
```

Comment:
why do we have `nullable=False` for only this field @tugbaalparslan ?

Reply:
I tried to see how it works by restricting it thru SqlAlchemy. I already restricted the rest of the fields through parser.